src/main/scala/beam/analysis/carridestats/CarRideStatsAreaBoundBox.scala

```diff
@@ -0,0 +1,12 @@
+package beam.analysis.carridestats
+
+case class CarRideStatsPoint(x: Double, y: Double) {
+  override def toString: String = s"($x,$y)"
+}
+
+case class CarRideStatsFilterAreaBoundBox(topLeft: CarRideStatsPoint, bottomRight: CarRideStatsPoint)
```

Comment:
I think you don't need all of these. There is already a usage of so called `Envelope` type from jts. The usage can be found, for example, in ZonalParkingManager: https://github.com/LBNL-UCB-STI/beam/blob/0dd3a931369a53f8b1930daf4213a9592783042e/src/main/scala/beam/agentsim/infrastructure/ZonalParkingManager.scala#L254